doc/sphinx/source/glossary.rst

```diff
@@ -33,6 +33,12 @@ Glossary
       The Intel nGraph library uses a function graph to represent an
       ``op``'s parameters and results.
 
+   fusion
+   
+      Fusion is the fusing, combining, merging, collapsing, or refactoring
+      of a graph's functional operations (``ops``) into one or more of
```

Comment:
Because we used to call the nGraph graph the "function graph" and "computational graph", which is confusing... you're right though. I will clean up this def.